src/sumaDeArraysCFunciones.ts

```diff
@@ -1,30 +1,27 @@
 let array1: number[] = new Array(6);
 let array2: number[] = new Array(6);
 let sumaDeArrays: number[] = new Array(0);
-let nums1 = 0;
-let nums2 = 0;
-function sumarArrays(nums1: number, nums2: number) {
-  for (let i = 0; i < 6; i++) {
-    nums1 = Number(
-      prompt("Ingreso numero: " + (i + 1) + "\nIngrese 6 numeros: ")
-    );
-    array1[i] = nums1;
-  }
-  console.log(array1);
-  for (let i = 0; i < 6; i++) {
-    nums2 = Number(
-      prompt("Ingreso numero: " + (i + 1) + "\nIngrese 6 numeros a sumar: ")
-    );
-    array2[i] = nums2;
-  }
-  console.log(array2);
-  let suma = 0;
+
+for (let i = 0; i < 6; i++) {
+  array1[i] = Number(
+    prompt("Ingreso numero: " + (i + 1) + "\nIngrese 6 numeros: ")
+  );
+}
+console.log(array1);
+for (let i = 0; i < 6; i++) {
+  array2[i] = Number(
+    prompt("Ingreso numero: " + (i + 1) + "\nIngrese 6 numeros a sumar: ")
+  );
+}
+console.log(array2);
+

```

Comment:
Los for que se usan para que el usuario agregue los valores de los arrays deben ir por fuera de la funcion, ya que no es parte de la suma.